lib/active_shipping/carriers/canada_post_pws.rb

```diff
@@ -776,17 +772,8 @@ def shipping_date(options)
       DateTime.strptime((options[:shipping_date] || Time.now).to_s, "%Y-%m-%d")
     end
 
-    def sanitize_location(location)
-      location_hash = location.is_a?(Location) ? location.to_hash : location
```

Comment:
The old version looks like location could be an instance of `Location` or a hash. If it is a hash, the new version if I'm not mistaken returns `nil` since a hash does not respond to `postal_code`. Is this intended?